### DISCUSSION TITLE
Audio Pulseaudio missing sample rate 11025

### DISCUSSION THREAD

**Discussion by NicolasGagne** [AlexandreRouma/SDRPlusPlus#554] [Q&A] [answered]
I follow the step describe here to fix the "underrated" error. 
https://github.com/AlexandreRouma/SDRPlusPlus/discussions/112
Now the only audio option I have is Pulseaudio, I don't mind but for the fact that it dosen't have the sample rate that I need: 11025. 
(It for WxtoIMG, witch is really picky on that)

For info I am on Ubuntu 20.04LTS.
I install the follwing package: sdrpp_ubuntu_focal_amd64.deb

![image](https://user-images.githubusercontent.com/28706249/146707820-ab03d8a5-29ad-43ae-a718-84b188b9e3c8.png)

can somebody help me?

**Comment by AlexandreRouma** [comment]
If the option doesn't show up, it means your sound driver/pulseaudio doesn't support it.

btw: I'd suggest using a more modern decoder for APT rather than WXtoImg which gives worse result and is very broken in general (eg. noaa-apt)

**Reply by NicolasGagne** [reply]
I don't belive that is the case, Intel integrated sound chip HDA is supporting 11025 (page 204)
https://www.intel.com/content/dam/www/public/us/en/documents/product-specifications/high-definition-audio-specification.pdf

some how, the driver or Alsa is removing this option, still looking.

**Comment by NicolasGagne** [comment, marked as answer]
In the end, I went a different route, I went in the source code and "Force" the 11025 rate in audio_sink/pulseaudio_sink/new_pulseaudio_sink main.cpp file. 

I will test for the following days and report back

**Reply by NicolasGagne** [reply]
Work perfectly for me, see attach files. 
[Modified-11025.zip](https://github.com/AlexandreRouma/SDRPlusPlus/files/7867013/Modified-11025.zip)
